### DISCUSSION TITLE
What is the expansion / meaning of "mch" in platform-specific code?

### DISCUSSION THREAD

**Discussion by dvogel** [vim/vim#15328] [General]
I've been hacking on vim for a few years and I've always accepted that the `_mch_` stem in function names is associated with platform-specific code. e.g. [when gui.c calls into platform-specific GUI implementation](https://github.com/vim/vim/blob/master/src/gui.c#L1335). I've always wondered what it means though. Does anyone here know? Is it initials of something?

**Comment by zeertzjq** [comment]
"machine"
